mods/beds/functions.lua

```diff
@@ -34,7 +36,7 @@ local function check_in_beds(players)
 		end
 	end
 
-	return #players > 0
+	return true
```

Comment:
You should change this back. If there is a merge conflict, don't override, but merge it manually.